neptune/codelist.go

```diff
@@ -210,3 +220,23 @@ func (n *NeptuneDB) GetCode(ctx context.Context, codeListID, edition string, cod
 		Code: code,
 	}, nil
 }
+
+// convert a flat array of record values into  a 2d array of records
+func createRecords(values []string, stride int) ([][]string, error) {
+	var records = [][]string{}
+	if len(values)%stride != 0 {
+		return nil, errors.New(fmt.Sprintf("List length is not divisible by %d", stride))
+	}
+	for i := 0; i < len(values); i += stride {
+
+		var record []string
+
+		for j := 0; j < stride; j++ {
+			value := values[i+j]
```

Comment:
No problem here I just want to make sure I follow:

If this skipping columns? If stride is 2 is it returning every other column?

Reply:
I have renamed `stride` to `valuesPerRecord`. Hopefully that makes it more clear. The database query returns a flat array of values, and those values need to be grouped into individual records. i.e. it will take the first two values for record 1, the next 2 values for record 2 etc